test/multitenancy/loginMethods.test.js

```diff
@@ -0,0 +1,274 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+const { printPath, setupST, startSTWithMultitenancy, killAllST, cleanST } = require("../utils");
+let assert = require("assert");
+const express = require("express");
+const request = require("supertest");
+let { Querier } = require("../../lib/build/querier");
+let { ProcessState } = require("../../lib/build/processState");
+const SuperTokens = require("../../");
+const Multitenancy = require("../../recipe/multitenancy");
+const EmailPassword = require("../../recipe/emailpassword");
+const ThirdPartyEmailPassword = require("../../recipe/thirdpartyemailpassword");
+const Passwordless = require("../../recipe/passwordless");
+const MultifactorAuth = require("../../recipe/multifactorauth");
+const Session = require("../../recipe/session");
+let { middleware, errorHandler } = require("../../framework/express");
+
+function compareWithoutOrdering(arr1, arr2) {
+    arr1.sort();
+    arr2.sort();
+    return JSON.stringify(arr1) === JSON.stringify(arr2);
```

Comment:
I'd prefer if this was done using [new Set](https://github.com/supertokens/supertokens-node/blob/1796f8c9503b954ed014b33d8295f934d26facc8/test/framework/fastify.test.js#L1236), it gives a much cleaner output